test/db/cmd/cmd_idp

```diff
@@ -24,148 +25,16 @@ EXPECT=<<EOF
 0x00401000  2  .text  void __cdecl SomeCoolFunction(void)
```

Comment:
Any particular reason why this is broken? It is not a class or a bitfield, should be supported by the current types model

Reply:
This is still marked as BROKEN.